profiles/coreos/base/package.accept_keywords

```diff
@@ -98,6 +98,8 @@ dev-util/checkbashisms
 
 =dev-libs/libgcrypt-1.9.4 ~amd64 ~arm64
 
+=dev-libs/libgcrypt-1.9.3 ~amd64
```

Comment:
Not sure why we need to add this line...?
https://github.com/flatcar-linux/portage-stable/tree/2b4cec8a3b9eeba789622aa46b969cd552697c4e/dev-libs/libgcrypt has only 1.9.4, not 1.9.3.
You would probably want to sync your portage-stable tree, or rebase https://github.com/flatcar-linux/portage-stable/pull/231 on top of main branch.

Reply:
I rebased my [#231](https://github.com/flatcar-linux/portage-stable/pull/231) on top of my main branch.
```
~/trunk/src/third_party/portage-stable $ git log --oneline
1550c51b1a2 (HEAD -> aniruddha/update-logrotate-3.18.1) Remove app-admin/logrotate from portage-stable
2b4cec8a3b9 (m/flatcar-master, github/main, main) Merge pull request #235 from flatcar-linux/dongsu/delete-keywords
6aeeefbc859 Merge pull request #241 from flatcar-linux/krnowak/update-cryptsetup-deps
```

But still the local build is showing the 
```
!!! All ebuilds that could satisfy "dev-libs/libgcrypt:0" for /mnt/host/source/src/build/images/amd64-usr/developer-2021.09.28+dev-main-nightly-3681+2021-11-06-0619-a1/rootfs/ have been masked.
!!! One of the following masked packages is required to complete your request:
- dev-libs/libgcrypt-1.9.3::portage-stable (masked by: ~amd64 keyword)
```

Reply:
Your trees look already ok, both coreos-overlay and portage-stable.

The local build failure you are seeing could happen because of remaining binary packages that still rely on libgcrypt 1.9.3.
The simplest way to get around the issue is to delete /build/amd64-usr completely, and start from scratch, i.e. `setup_board`, `build_packages`, etc.

Anyway I have started a CI build for the PRs. http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/4098/cldsv
Will see if it passes.

Reply:
Thanks! 
I will start building the latest alpha SDK from tomorrow. Hopefully that will fix all issue.

Reply:
CI passed. http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/4098/cldsv
Will let @tormath1 & @krnowak have another look.